### PR TITLE
DCOS-7818: Hide sidebar logo in short viewports

### PR DESCRIPTION
**Pull request by jfurrow**

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @philipnrmn

File: src/styles/layout/sidebar.less [hunk 1]
```diff
@@ -903,6 +903,23 @@ Sidebar Open
...
+      &.is-expanded {
+
+        .sidebar-content {
+
+          .sidebar-logo-container {
+            display: none;
+          }
+        }
+      }
+    }
```
Comment (kennyt) [inline src/styles/layout/sidebar.less]: spacing is weird here. these closing braces don't have newlines between them but the opening ones do

Comment (jfurrow) [inline src/styles/layout/sidebar.less]: Done!

**Comment by kennyt** [comment]
works great!

**Comment by philipnrmn** [comment]
Tests passing locally.

**Pull request merged**